project_templates/test_report/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}/.github/workflows/docgen_from_Jira.yaml

```diff
@@ -0,0 +1,61 @@
+name: docgen from Jira
+on:
+  workflow_dispatch
+
+jobs:
+   generate:
+     runs-on: ubuntu-latest
+     steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # full history for metadata
+          submodules: false
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - name: Python install
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install "lander<2.0.0"
+
+      - name: condaenv
+        run: |
+          conda init bash
+          conda create --name docsteady-env docsteady -c lsst-dm -c conda-forge
+        shell: bash
```

Comment:
Since miniconda is already available, I believe you can remove this step and replace the `source activate` line in the next step with:
```
# Install into the base environment
conda install -c lsst-dm -c conda-forge docsteady
```

Reply:
i tried that originally and could not get it to work - it said the shell was not initiailised for conda so I ended up wit this

Reply:
OK, but you still shouldn't need to create a new environment.  Installing into the base environment should be fine.